CommandHelper/LocalPackages/commands/chars/saberarturiapendragon.yml

```diff
@@ -0,0 +1,23 @@
+maxHearts: 80
+buffs:
+- /effect <plr> 5 9999 15 true
+- /effect <plr> 11 9999 3 true
+- /give <plr> minecraft:golden_sword 1 16:20 name:&6E&ex&6c&ea&6l&ei&6b&eu&6r
+- /shot give <plr> excaliburbeam
+chatcolor: '&9'
```

Comment:
name and chatcolor have been moved to 'costumes', check out beast boy as reference